dashboard/app/models/ability.rb

```diff
@@ -89,7 +89,7 @@ def initialize(user)
         can :manage, Workshop do |workshop|
           workshop.facilitators.include? user
         end
-        can [:read, :start, :end], Pd::Workshop, facilitators: {id: user.id}
+        can [:read, :start, :end, :workshop_survey], Pd::Workshop, facilitators: {id: user.id}
```

Comment:
Sure it worked just for facilitator? If you're logged in as admin, then you have manage access to `Pd::Workshop` which covers all actions.

This is probably worth adding unit tests for access, like in the [workshops_controller_test](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/controllers/ops/workshops_controller_test.rb)